tools/ARIAtools/ARIAProduct.py

```diff
@@ -221,6 +222,16 @@ def __continuous_time__(self):
         if len(self.products)==1:
             sorted_products.extend([[dict(zip(self.products[0][0].keys(), [list(a) for a in zip(self.products[0][0].values())])), dict(zip(self.products[0][1].keys(), [list(a) for a in zip(self.products[0][1].values())]))]])
 
+        # Check for (and remove) duplicate products
+        for i, j in enumerate(self.products[:-1]):
+            # If scenes share >90% spatial overlap AND same dates, they MUST be duplicates. Reject the latter.
+            if (self.products[i+1][0]['pair_name'][9:]==j[0]['pair_name'][9:]) and (self.products[i+1][0]['pair_name'][:8]==j[0]['pair_name'][:8]) and (open_shapefile(self.products[i+1][1]['productBoundingBox'], 'productBoundingBox', 1).intersection(open_shapefile(j[1]['productBoundingBox'], 'productBoundingBox', 1)).area)/(open_shapefile(j[1]['productBoundingBox'], 'productBoundingBox', 1).area)>0.9:
+                print("WARNING: Duplicate date captured. Rejecting scene %s"%(self.products[i+1][1]['unwrappedPhase'].split('"')[1]))
```

Comment:
Would not say a duplicate date, rather say "duplicate product", as we have many products with he same date.

Reply:
Ok, committed in.